src/tools/miri/tests/fail/layout_cycle.stderr

```diff
@@ -1,5 +1,6 @@
 error[E0391]: cycle detected when computing layout of `S<S<()>>`
    |
+   = note: see https://rustc-dev-guide.rust-lang.org/overview.html#queries and https://rustc-dev-guide.rust-lang.org/query.html for more information
    = note: ...which requires computing layout of `<S<()> as Tr>::I`...
    = note: ...which again requires computing layout of `S<S<()>>`, completing the cycle
```

Comment:
is there some way to put this at the end of the cycle error instead of the beginning? the order is meaningful here and it seems confusing to interrupt the cycle with the link.
```suggestion
   = note: ...which requires computing layout of `<S<()> as Tr>::I`...
   = note: ...which again requires computing layout of `S<S<()>>`, completing the cycle
   = note: see https://rustc-dev-guide.rust-lang.org/overview.html#queries and https://rustc-dev-guide.rust-lang.org/query.html for more information
```